next.config.js

```diff
@@ -2,6 +2,10 @@ const withSvgr = require("next-plugin-svgr");
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  webpack: (config) => {
```

Comment:
this is some more crap from walletconnect. they seem to have somehow made the nextjs compiler think that these dependencies should be bundled in our project. see https://github.com/WalletConnect/walletconnect-monorepo/issues/1908